components/Forms/FormAddPlaces.js

```diff
@@ -0,0 +1,170 @@
+import styled from "styled-components";
+import React from "react";
+import StyledTertiarySection from "../Sections/StyledTertiarySection";
+
+import { useState } from "react";
+import useSWR from "swr";
+
+export default function AddPlaces({ getUniqueValues }) {
+  const [isChecked, setIsChecked] = useState([]);
+
+  const { data, error, isLoading } = useSWR("/api/activities");
```

Comment:
are you fetching real activities here or just the values for the dropdown?
I think it's faster and therefore better to get the values for the dropdown from a local file.
If you are indeed loading more than just values/labels here, I would argue it might not be necessary in a form component! :-)